src/components/SearchBar/SearchBarStyled.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import { baseColors, basePadding, fontSizing, transition } from "#/theme";
+
+export const SearchInput = styled.input`
+  margin: ${basePadding.medium} 0;
+  padding: ${basePadding.medium};
+  width: 100%;
+  max-width: 1200px;
+  border: 1px solid ${baseColors.gray1};
+  border-radius: 4px;
+  outline: none;
+  font-size: ${fontSizing.base};
+  color: ${baseColors.white};
+  transition: ${transition.default};
+
```

Comment:
This is super nit-picky, but I will eventually alphabetize this if you don't, lol.